invenio_rdm_migrator/streams/oauth/transform.py

```diff
@@ -7,21 +7,19 @@
 
 """Invenio RDM migration record transform interfaces."""
 
-from abc import abstractmethod
-
 from ...transform import EncryptMixin, IdentityDictKeyMixin, Transform
 
 
-class OAuthServerTokenTransform(Transform, EncryptMixin, IdentityDictKeyMixin):
+class OAuthServerTokenTransform(EncryptMixin, Transform, IdentityDictKeyMixin):
```

Comment:
I guess this helps with not having to initiate the encryption/decryption engines or specify the super init. It might be handy to apply it to some of the user transforms (in ZenodoRDM IIRC).